inputs/manager-inputs.yaml

```diff
@@ -436,6 +436,14 @@ inputs:
     type: integer
     default: 2
 
+  gate_keeper_bucket_size:
```

Comment:
Why keep this uninformative name? what about `max_parallel_operations_per_deployment` ?